website/templates/project/settings.mako

```diff
@@ -474,6 +497,7 @@
       window.contextVars.wiki.isEnabled = ${wiki.short_name in addons_enabled | sjson, n };
       window.contextVars.currentUser = window.contextVars.currentUser || {};
       window.contextVars.currentUser.institutions = ${ user['institutions'] | sjson, n };
+      window.contextVars.user = ${ user | sjson, n };
```

Comment:
In this context, there is no difference. However, sharing.js, which is used both for contributor and private links, expects a user to be set in the context vars,
https://github.com/CenterForOpenScience/osf.io/pull/7335/files#diff-fd51466e5d40043598e7fe4e21bd0100
I can simply set window.contextVars.user = window.contextVars.currentIUser if that's better